_includes/payment-order-unscheduled.md

```diff
@@ -0,0 +1,82 @@
+{% capture features_url %}{% include documentation-section-url.md href='/features' %}{% endcapture %}
+
+## Unscheduled Purchase
+
+An `unscheduled purchase`, also called a Merchant Initiated Transaction (MIT),
+is a payment which uses a `unscheduledToken` generated through a previous payment in
+order to charge the same card at a later time. They are done by the merchant
+without the cardholder being present.
```

Comment:
I'll ask Per M if he has a good explanation on the difference between unscheduled and recur in line with the new guidelines from Acquiring. If so, this text might be changed a bit.